schema/sysmon.schema

```diff
@@ -558,3 +558,153 @@ type sysmon.RawAccessRead = record {
     Device: string,
 }
 
+// The following beats type definitions wrap the above raw sysmon events.
+// They are effectively a carrier of the Sysmon data that comes with some
+// additional meta data.
+//
+// The logstash filter configuration for the below defined Winlogbeat events
+// looks like this:
```

Comment:
```suggestion
// An example logstash filter configuration that generates `meta_data` and
// `event_data` fields in the JSON output could look like this:
```

The logstash config provided here is not the only valid one that can be used with this schema.